tema4/index.js

```diff
@@ -0,0 +1,7 @@
+const addNumber = (...arguments) => {
+    return arguments.reduce((sum, numbers) => sum + numbers, 0);
```

Comment:
```suggestion
    return arguments.reduce((sum, number) => sum + number, 0);
```
Sa denumim corect parametri - asa cum sunt ei. Acesta este un singur numar, din lista de argumente. Argumentele ar fi putut sa se cheme "numbers" dar in interiorul functiei "reduce" fiecare argument e un singur numar